artifacts/examples/osm-canary-steps.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: flagger.app/v1beta1
+kind: Canary
+metadata:
+  name: podinfo
+  namespace: test
+spec:
+  provider: osm
+  targetRef:
+    apiVersion: apps/v1
+    kind: Deployment
+    name: podinfo
+  autoscalerRef:
```

Comment:
Remove the hpa refs from all tests please.

Reply:
Removed.